ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py

```diff
@@ -717,3 +717,40 @@ def test_jax_lax_bitwise_and(
         x=np.asarray(x[0], dtype=input_dtype[0]),
         y=np.asarray(x[1], dtype=input_dtype[1]),
     )
+
+
+# bitwise_or
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=ivy_jax.valid_int_dtypes,
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+    as_variable=helpers.list_of_length(x=st.booleans(), length=2),
```

Comment:
please remove this line as it is handled by the `@handle_cmd_line_args` decorator

Reply:
done